catalog/USFS/USFS_GTAC_TreeMap_v2016.jsonnet

```diff
@@ -0,0 +1,2351 @@
+local id = 'USFS/GTAC/TreeMap/v2016';
+local latest_id = id;
+local predecessor_id = '';
+local subdir = 'USFS';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local latest_basename = std.strReplace(latest_id, '/', '_');
+local predecessor_basename = std.strReplace(predecessor_id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+
+{
+  // TODO(schwehr): Remove when the dataset is ready.
+  'gee:skip_indexing': true,
+
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'USFS TreeMap v2016 (Conterminous United States)',
+  version: 'v2016',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    This product is part of the TreeMap data suite. It provides detailed spatial information on forest characteristics including
+    number of live and dead trees, biomass, and carbon across the entire forested extent of the continental United States in 2016.
+
+    TreeMap products are the output of a random forests machine learning algorithm that assigns the most similar Forest Inventory Analysis (FIA)
+    plot to each pixel of gridded LANDFIRE input data. The objective is to combine the complimenting strengths of detailed-but-spatially-sparse
+    FIA data with less-detailed-but-spatially-comprehensive LANDFIRE data to produce better estimations of forest characteristics at a variety of scales.
+    TreeMap is being used in both the private and public sectors for projects including fuel treatment planning, snag hazard mapping, and estimation of
+    terrestrial carbon resources.
+
+    TreeMap is distinct from other imputed forest vegetation products in that it provides an FIA plot identifier to each pixel whereas other datasets
+    provide forest characteristics such as live basal area (e.g., Ohmann and Gregory 2002; Pierce Jr et al. 2009; Wilson, Lister, and Riemann 2012).
+    The FIA plot identifier can be linked to the hundreds of variables and attributes recorded for each tree and plot in the FIA DataMart, FIA's public
+    repository of plot information (Forest Inventory Analysis 2022a).
+
+    TreeMap v2016 contains one image, a 22-band 30 x 30m resolution gridded map of the forests of the continental United States circa 2016, with each band 
```

Comment:
I'd put this paragraph higher up, as it's important, but people might stop reading before they get all they way here.